### ISSUE TITLE
Create PPA

### ISSUE THREAD

**Issue by aircrack-ng** [aircrack-ng/aircrack-ng#1590]
**Reported by misterx on 18 Jun 2015 00:32 UTC**

Create a PPA repository.

See https://help.launchpad.net/Packaging/PPA and https://askubuntu.com/questions/71510/how-do-i-create-a-ppa

**Comment by aircrack-ng** [comment]
**Comment by misterx on 18 Jun 2015 00:32 UTC**

See #1561.

**Comment by PranavBhattarai** [comment]
I wonder when PPA is going to be available.
Is there a reason why **sudo apt install aircrack-ng** don't install the latest stable aircrack-ng in the OS? @jbenden 

```

Bond@JamesBond ~  aircrack-ng --help

  Aircrack-ng 1.5.2  - (C) 2006-2018 Thomas d'Otreppe
  https://www.aircrack-ng.org


```
Distro: **Ubuntu 19.10**

**Comment by aircrack-ng** [comment]
Not sure if we will ever do a PPA but we do have packages for a number of distro: https://aircrack-ng.blogspot.com/2018/10/aircrack-ng-packages.html

TL;DR: We are using packagecloud.io, and if you head to host them and we offer 2 channels: [release](https://packagecloud.io/aircrack-ng/release), which only contains the releases, and [git](https://packagecloud.io/aircrack-ng/git) that contains every commit that builds successfully.

**Comment by PranavBhattarai** [comment]
@aircrack-ng well I tried to install the release version from the link you mentioned but it didn't work.

As you can see:
`
Bond@JamesBond ~ curl -s https://packagecloud.io/install/repositories/aircrack-ng/release/script.deb.sh | sudo zsh 
[sudo] password for pranav: 
Detected operating system as Ubuntu/eoan.
Checking for curl...
Detected curl...
Checking for gpg...
Detected gpg...
Running apt-get update... done.
Installing apt-transport-https... done.
Installing /etc/apt/sources.list.d/aircrack-ng_release.list...done.
Importing packagecloud gpg key... done.
Running apt-get update... done.

The repository is setup! You can now install packages.

Bond@JamesBond ~ sudo apt update --fix-missing
Err:15 https://packagecloud.io/aircrack-ng/release/ubuntu eoan Release
  404  Not Found [IP: 55.265.243.154 453]
Hit:16 http://np.archive.ubuntu.com/ubuntu eoan-updates InRelease
Hit:17 http://np.archive.ubuntu.com/ubuntu eoan-backports InRelease
Reading package lists... Done
E: The repository 'https://packagecloud.io/aircrack-ng/release/ubuntu eoan Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
`

**Comment by 84rn** [comment]
@PranavBhattarai you need to modify the .list file provided by packagecloud.io so it refers to any/any and not your system os/dist. Try this:
```
/etc/apt/sources.list.d/aircrack-ng_release.list
------------------------------------------------


deb [signed-by=/usr/share/keyrings/aircrack-ng_release-archive-keyring.gpg] https://packagecloud.io/aircrack-ng/release/any/ any main

deb-src [signed-by=/usr/share/keyrings/aircrack-ng_release-archive-keyring.gpg] https://packagecloud.io/aircrack-ng/release/any/ any main
```

@aircrack-ng maybe make this more clear in the docs.
